app/presenters/tag_presenter.rb

```diff
@@ -0,0 +1,29 @@
+#   Copyright (c) 2016, Diaspora Inc.  This file is
+#   licensed under the Affero General Public License version 3 or later.  See
+#   the COPYRIGHT file.
+
+class TagPresenter < BasePresenter
```

Comment:
`TagPresenter` to me sounds like being for a single `Tag`, how about `TagStreamPresenter`?